test/tests/unit/backend.ts

```diff
@@ -1,15 +1,21 @@
 import chai from "chai";
+import { redisClient } from "../../../backend/redis";
 const v1 = require("../../../backend/lumens");
 const v2v3 = require("../../../backend/v2v3/lumens");
 
 describe("lumens v1", function () {
+  this.timeout(10000);
```

Comment:
these added test timeouts are not related to redis but were added bc occasionally calls to Horizon would take longer than usual (assuming traffic related) causing a timeout error (default is 2s). So 10s seemed to be a good timeout empirically

Reply:
Do you mind adding a comment at the top of this file that explains this?

Reply:
👍 https://github.com/stellar/dashboard/pull/205/commits/7beea1a978ada649f6bf18bc0d0bd17388269079